internals/webpack/webpack.dev.babel.js

```diff
@@ -53,6 +53,10 @@ module.exports = require('./webpack.base.babel')({
 
   // Emit a source map for easier debugging
   devtool: 'cheap-module-eval-source-map',
+
+  performance: {
+    hints: false,
```

Comment:
`performance: {hints: false` doesn't seem to be working:
![image](https://cloud.githubusercontent.com/assets/1481063/21743624/f4f12e8c-d4ba-11e6-9f55-5303eae38ad7.png)

This will certainly create additional support issues.